CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## v6.0.1 - 2018-01-19
+
+### Engine updates and fixes
+
+- Update PowerShell to use `2.0.5` dotnet core runtime and packages. (#5903, #5961) (Thanks @iSazonov!)
```

Comment:
One more update needs to be added for #5961

Reply:
What additional is needed for 5961?

Reply:
Never mind. I didn't see the `#5961` was mentioned.